tests/src/test/scala/system/basic/WskRuleTests.scala

```diff
@@ -371,4 +371,24 @@ class WskRuleTests
             }
     }
 
+    it should "disable a rule and check its status is displayed when listed" in withAssetCleaner(wskprops) {
+        (wp, assetHelper) =>
+            val ruleName = withTimestamp("ruleDisable")
+            val ruleName2 = withTimestamp("ruleEnable")
+            val triggerName = withTimestamp("ruleDisableTrigger")
+            val actionName = withTimestamp("ruleDisableAction")
+
+            ruleSetup(Seq(
+                (ruleName, triggerName, (actionName, actionName, defaultAction)),
+                (ruleName2, triggerName, (actionName, actionName, defaultAction))),
+                assetHelper)
+
+            wsk.rule.disable(ruleName)
+            wsk.rule.list().stdout should include(ruleName)
+            wsk.rule.list().stdout should include(ruleName2)
+            wsk.rule.list().stdout should include("active")
```

Comment:
Make sure the active and inactive statuses are connected to the correct rule names